### PR TITLE
1427 remove email and password from settings because login gov

### PR DESCRIPTION
**Pull request by jonathanbobel**
## Description

Removing Email and Password from settings because that is now being handled by Login.gov

### CODE REVIEW

**Review by terrazoon — APPROVED** [review]
LGTM

**Review by ccostino — COMMENTED** [review]
Thanks, @jonathanbobel!  I have one quick question on the email address removal that I'm asking @em-herrick or @maomeara63 to weigh in on.

File: app/templates/views/user-profile.html [hunk 1]
```diff
@@ -27,21 +27,6 @@ <h1 class="font-body-2xl margin-bottom-3">User profile</h1>
...
-        {% if can_see_edit %}
-          {{ edit_field(
-              'Change',
-              url_for('.user_profile_email'),
-              suffix='email address'
-            )
-          }}
-        {% else %}
-          {{ text_field('') }}
-        {% endif %}
```
[the remark this answers is not in the document]
Comment (ccostino) [reply app/templates/views/user-profile.html:42]: Resolved in Slack!  https://github.com/GSA/notifications-admin/pull/1399 has what we want for this.

**Review by ccostino — APPROVED** [review]
Thanks, @jonathanbobel!

**Pull request merged**